region/maridia/outer/Fish Tank.json

```diff
@@ -891,21 +891,43 @@
       "entranceCondition": {
         "comeInRunning": {
           "speedBooster": true,
-          "minTiles": 0.4375
+          "minTiles": 4.4375
         }
       },
       "requires": [
         "canWaterShineCharge",
         {"or": [
           {"shinespark": {"frames": 41, "excessFrames": 7}},
           {"and": [
+            "canDodgeWhileShooting",
             "canMidairShinespark",
             {"shinespark": {"frames": 38, "excessFrames": 7}}
           ]}
         ]}
       ],
       "note": "Start the dash as close to the door transition as possible in order to have a larger window for storing the shinecharge."
     },
+    {
+      "link": [2, 5],
+      "name": "Suitless Stutter Water Shinecharge",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 0.4375
+        }
+      },
+      "requires": [
+        "canStutterWaterShineCharge",
```

Comment:
This strat should have the note from above, and maybe the one above should mention the tap. But I'm not really seeing a stutter as the correct tech, since there's not really space for that. 

BTW, it looks like you have 2-3 pixels of space to start, depending on subpixels